File: Javascript/TallerJS/Ejercicio6.js
// 7.Pedir un número y decir si es un número negativo, si es positivo o cero.

let numero = parseInt(prompt("Ingrese un numero"))

if(numero < 0){ 
    console.log("El numero ", numero, " es negativo")
}else if(numero > 0){
    console.log("El numero ", numero, " es positivo")
}else{
    console.log("El numero ", numero, " es cero")
}

File: Python/Condicionales/Ejercicio6.py
# 7.Pedir un número y decir si es un número negativo, si es positivo o cero.

numero = int(input("Ingrese un numero "))

if numero < 0 : 
    print("El numero ", numero, " es negativo")
elif numero > 0 :
    print("El numero ", numero, " es positivo")
else:
    print("El numero ", numero, " es cero")
